deepchecks/checks/performance/class_performance_imbalance.py

```diff
@@ -0,0 +1,236 @@
+                ))
+                for name, metric_func in self._get_default_metrics().items()
+            })
+
+        def display():
+            title = (
+                'Class Performance Imbalance Check for binary data'
+                if len(labels) == 2
+                else 'Class Performance Imbalance Check for multi-class data'
+            )
+
+            df.transpose().plot.bar(
+                title=title,
+                backend='matplotlib',
+                xlabel='Metrics',
+                ylabel='Values'
+            )
+
+        return CheckResult(
+            value=df.transpose().to_dict(),
+            header='Class Performance Imbalance',
+            check=type(self),
+            display=display
+        )
+
+    def _get_default_metrics(self) -> t.Dict[str, t.Callable[..., np.ndarray]]:
+        return {
+            'Accuracy': partial(recall_score, zero_division=0, average=None),
+            'Precision': partial(precision_score, zero_division=0, average=None),
+            'Recall': partial(recall_score, zero_division=0, average=None)
+        }
+
+    def _execute_alternative_metrics(
+        self,
+        y_true: pd.Series,
+        y_pred: pd.Series,
+    ) -> t.Dict[str, t.Dict[t.Hashable, t.Union[int, float]]]:
+        check_name = type(self).__name__
+        metrics = t.cast(t.Mapping[str, MetricFunc], self.alternative_metrics)
+        result: t.Dict[str, t.Dict[t.Hashable, t.Union[int, float]]] = {}
+
+        for metric_name, metric_func in metrics.items():
+            metric_result = metric_func(y_true, y_pred)
+
+            if not isinstance(metric_result, dict):
+                raise DeepchecksValueError(
+                    f'Check {check_name} expecting that alternative metrics will return '
+                    f"not empty instance of 'Dict[Hasable, float|int]', but got {type(metric_result).__name__}" #pylint: disable=inconsistent-quotes
+                )
+
+            if len(metric_result) == 0:
+                raise DeepchecksValueError(
+                    f'Check {check_name} expecting that alternative metrics will return '
+                    "not empty instance of 'Dict[Hasable, float|int]'" #pylint: disable=inconsistent-quotes
+                )
+
+            incorrect_values = [v for v in metric_result.values() if not isinstance(v, (int, float))]
+
+            if len(incorrect_values) != 0:
+                value_type = type(incorrect_values[0]).__name__
+                raise DeepchecksValueError(
+                    f'Check {check_name} expecting that alternative metrics will return '
+                    f"not empty instance of 'Dict[Hasable, float|int]', but got Dict[Hashable, {value_type}]" #pylint: disable=inconsistent-quotes
+                )
+
+            result[metric_name] = metric_result
+
+        return result
+
+    def add_condition_percentage_difference_not_greater_than(self: CP, threshold: float = 0.3) -> CP:
```

Comment:
Think it's better that the default condition be on ratio of change / difference rather than absolute value. Not saying that having this one in addition is bad.

Reply:
My bad! It is already percent change...